articles/backup/active-directory-backup-restore.md

```diff
@@ -18,7 +18,12 @@ This article outlines the proper procedures for backing up and restoring Active
 
 - Make sure at least one domain controller is backed up. If you back up more than one domain controller, make sure all the ones holding the [FSMO (Flexible Single Master Operation) roles](/windows-server/identity/ad-ds/plan/planning-operations-master-role-placement) are backed up.
 
-- Back up Active Directory frequently. The backup should never be more than the tombstone lifetime (by default 60 days), because objects older than the tombstone lifetime will be "tombstoned" and no longer considered valid.
+- Back up Active Directory frequently. The backup should never be more than the tombstone lifetime (TSL) because objects older than the tombstone lifetime will be "tombstoned" and no longer considered valid.
+  - The default tombstone lifetime for domains built on Windows Server 2003 SP2 and above is 180 days.
+  - The configured tombstone lifetime can be verified using the following PowerShell:
```

Comment:
- The configured TSL can be verified using the following PowerShell script/command:

_Comment: Use the appropriate one to make it sound complete._